GoTelemetry/server/pb/telemetry.pb.go

```diff
@@ -69,99 +240,574 @@ func (*TelemetryEvent) Descriptor() ([]byte, []int) {
+}
+
+func (x *TelemetryEvent_Telemetry_Gyro) Reset() {
+	*x = TelemetryEvent_Telemetry_Gyro{}
+	if protoimpl.UnsafeEnabled {
+		mi := &file_telemetry_proto_msgTypes[3]
+		ms := protoimpl.X.MessageStateOf(protoimpl.Pointer(x))
+		ms.StoreMessageInfo(mi)
+	}
+}
+
+func (x *TelemetryEvent_Telemetry_Gyro) String() string {
+	return protoimpl.X.MessageStringOf(x)
+}
+
+func (*TelemetryEvent_Telemetry_Gyro) ProtoMessage() {}
+
+func (x *TelemetryEvent_Telemetry_Gyro) ProtoReflect() protoreflect.Message {
+	mi := &file_telemetry_proto_msgTypes[3]
+	if protoimpl.UnsafeEnabled && x != nil {
+		ms := protoimpl.X.MessageStateOf(protoimpl.Pointer(x))
+		if ms.LoadMessageInfo() == nil {
+			ms.StoreMessageInfo(mi)
+		}
+		return ms
+	}
+	return mi.MessageOf(x)
+}
+
+// Deprecated: Use TelemetryEvent_Telemetry_Gyro.ProtoReflect.Descriptor instead.
+func (*TelemetryEvent_Telemetry_Gyro) Descriptor() ([]byte, []int) {
+	return file_telemetry_proto_rawDescGZIP(), []int{0, 2}
+}
+
+func (x *TelemetryEvent_Telemetry_Gyro) GetGyroX() float32 {
 	if x != nil {
 		return x.GyroX
 	}
 	return 0
 }
 
-func (x *TelemetryEvent) GetGyroY() float32 {
+func (x *TelemetryEvent_Telemetry_Gyro) GetGyroY() float32 {
 	if x != nil {
 		return x.GyroY
 	}
 	return 0
 }
 
-func (x *TelemetryEvent) GetGyroZ() float32 {
+func (x *TelemetryEvent_Telemetry_Gyro) GetGyroZ() float32 {
 	if x != nil {
 		return x.GyroZ
 	}
 	return 0
 }
 
+type TelemetryEvent_Telemetry_Location struct {
+	state         protoimpl.MessageState
+	sizeCache     protoimpl.SizeCache
+	unknownFields protoimpl.UnknownFields
+
+	Rows uint32  `protobuf:"varint,1,opt,name=rows,proto3" json:"rows,omitempty"`
+	Cols uint32  `protobuf:"varint,2,opt,name=cols,proto3" json:"cols,omitempty"`
+	Data []int32 `protobuf:"varint,3,rep,packed,name=data,proto3" json:"data,omitempty"`
+}
+
+func (x *TelemetryEvent_Telemetry_Location) Reset() {
+	*x = TelemetryEvent_Telemetry_Location{}
+	if protoimpl.UnsafeEnabled {
+		mi := &file_telemetry_proto_msgTypes[4]
+		ms := protoimpl.X.MessageStateOf(protoimpl.Pointer(x))
+		ms.StoreMessageInfo(mi)
+	}
+}
+
+func (x *TelemetryEvent_Telemetry_Location) String() string {
+	return protoimpl.X.MessageStringOf(x)
+}
+
+func (*TelemetryEvent_Telemetry_Location) ProtoMessage() {}
+
+func (x *TelemetryEvent_Telemetry_Location) ProtoReflect() protoreflect.Message {
+	mi := &file_telemetry_proto_msgTypes[4]
+	if protoimpl.UnsafeEnabled && x != nil {
+		ms := protoimpl.X.MessageStateOf(protoimpl.Pointer(x))
+		if ms.LoadMessageInfo() == nil {
+			ms.StoreMessageInfo(mi)
+		}
+		return ms
+	}
+	return mi.MessageOf(x)
+}
+
+// Deprecated: Use TelemetryEvent_Telemetry_Location.ProtoReflect.Descriptor instead.
+func (*TelemetryEvent_Telemetry_Location) Descriptor() ([]byte, []int) {
+	return file_telemetry_proto_rawDescGZIP(), []int{0, 3}
+}
+
+func (x *TelemetryEvent_Telemetry_Location) GetRows() uint32 {
+	if x != nil {
```

Comment:
Can you explain this a little more? Rows and columns from my understanding are in the 6x6 grid and this func returns 1 number. What does that number represent?